### PR TITLE
Implement keypoint matching

### LINKED ISSUE #32: Replace first demo-map with one which is human friendly and have enough information for key-points selection.

### PR DESCRIPTION
**Pull request by olegboev**

### CODE REVIEW

**Review by mjh-sakh — CHANGES_REQUESTED** [review]
Please delete env_1d from PR. There is a big change to it in the PR 35 submitted earlier and I don't want to spend time resolving conflicts. 
I got point about cv2.

File: Stage1/main.py [hunk 1]
```diff
@@ -1,11 +1,46 @@
...
+        scan = np.expand_dims(scan, axis=0)
+
+        kp, des = detector.detect_and_compute(scan)
+        frame_curr = Frame(scan, kp, des)
+
+        matches = list()
+        if frame_curr.descriptors and frame_prev.descriptors:
+            matches = matcher.match(np.array(frame_prev.descriptors), np.array(frame_curr.descriptors))
+
+        image_output = draw_pland_and_frames(plan_with_scan, frame_prev, frame_curr, matches)
```
Comment (mjh-sakh) [inline Stage1/main.py:35]: Where is def for draw_pland_and_frames?

File: Stage1/main.py [hunk 2]
```diff
@@ -1,11 +1,46 @@
...
 
-plan = load_plan(r'environment/demo_map_1.jpg')
+        cv2.imshow('map', image_output)
+        k = cv2.waitKey(25)
+        if k == 27:
```
Comment (mjh-sakh) [inline Stage1/main.py]: Please add comment on what is key 27 is. Space?

**Review by mjh-sakh — APPROVED** [review]

File: Stage1/main.py (hunk 1 above)
Comment (olegboev) [reply Stage1/main.py:35]: See [env_1d.py](https://github.com/mjh-sakh/Project-bI/pull/39/commits/c25b7473fff52254b5d2147b5cd31e03f8e7abef#diff-f6a41ba02203b089e0ffe8ccd261599358dd2c692820ebc2ec809f56386cf97e)

**Pull request merged**